### PR TITLE
AFD 78 Bookshelf View

### PR DESCRIPTION
**Pull request by TheGuardingDark**
Re-built bookshelf view for react. Styled and added breakpoints.
Cards are hard-coded for now, but will be populated from books in database later

### CODE REVIEW

File: src/styles/_card.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            }
+        }
+    }
+    @media only screen and (max-width: 480px) {
+        width: 22rem;
+    }
+    @media only screen and (max-width: 375px) {
+        width: 19rem;
+    }
+}
```
Comment (katieburrows) [inline src/styles/_card.scss:40]: There is no newline at end of file.

File: src/styles/_mobileSearch.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        width: 114%;
+        &__input {
+            height: 2.5rem;
+        }
+        &__icon {
+            top: -3.7rem;
+        }
+    }
+
+}
```
Comment (katieburrows) [inline src/styles/_mobileSearch.scss:37]: No newline at end of file.

File: src/styles/index.scss [hunk 3]
```diff
@@ -157,4 +159,46 @@ body {
...
+      grid-column: auto;
+      font-size: 2rem;
+      text-align: center;
+    }
+  }
+  .mobileSearch {
+    @include ms.mobileSearch;
+  }
+}
+
```
Comment (katieburrows) [inline src/styles/index.scss:204]: No newline at end of file.

**Review by katieburrows — APPROVED** [review]
Please note comments where no newline appears at the end of the file.

File: src/styles/_card.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
+@use "./variables" as v;
+@use "./mixins" as m;
+
+
```
Comment (aperezsantos) [inline src/styles/_card.scss:4]: Just a heads up about this extra line here

File: src/styles/index.scss [hunk 5]
```diff
@@ -3,6 +3,8 @@
 @use "./nav" as n;
 @use "./hamburger" as h;
 @use "./footer" as f;
+@use "./card" as c;
+@use "./mobileSearch" as ms;
```
Comment (aperezsantos) [inline src/styles/index.scss:9]: Another extra line

**Review by aperezsantos — APPROVED** [review]
Awesome continued work on the project. Very clean and concise.

File: src/components/BookCard.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Jacket from '../styles/images/snuff.jpg';
+
+const BookCard = ({ src, title, author }) => {
+  return (
+    <div className="card">
+      <Link to="/details">
```
Comment (scullenDev) [inline src/components/BookCard.jsx:8]: Note that eventually this link should include the book id at the end, ie `/details/7` but for now this is fine until you make that dynamic.

**Review by scullenDev — APPROVED** [review]
Looks good!

**Pull request merged**